packages/manager/apps/hpc-vmware-managed-vcd/src/tracking.constant.ts

```diff
@@ -1,20 +1,144 @@
+import {
+  ButtonType,
+  PageLocation,
+  TrackingClickParams,
+} from '@ovh-ux/manager-react-shell-client';
+
 export const LEVEL2 = {
   EU: {
     config: {
-      level2: '56',
+      level2: '120',
     },
   },
   CA: {
     config: {
-      level2: '56',
+      level2: '120',
     },
   },
   US: {
     config: {
-      level2: '56',
+      level2: '120',
     },
   },
 };
-export const UNIVERSE = 'HostedPrivatedCloud';
+export const UNIVERSE = 'Enterprise';
 export const SUB_UNIVERSE = 'HostedPrivatedCloud';
-export const APP_NAME = 'hpc-vmware-managed-vcd';
+export const APP_NAME = 'managed_vcd';
+
+type GetTrackingParams = (param: string | string[]) => TrackingClickParams;
+type AppTracking = Record<
+  string,
+  Record<string, TrackingClickParams | GetTrackingParams>
+>;
+
+const definedTrackingConfig = <T extends AppTracking>(config: T) => config;
+
+export const TRACKING = definedTrackingConfig({
+  onboarding: {
+    guideClick: (guideName: string): TrackingClickParams => ({
+      location: PageLocation.page,
+      buttonType: ButtonType.externalLink,
+      actionType: 'navigation',
+      actions: [`go-to-${guideName}`],
+    }),
+  },
+  listing: {
+    details: {
+      location: PageLocation.datagrid,
+      buttonType: ButtonType.button,
+      actionType: 'navigation',
+      actions: [`details_${APP_NAME}`],
+    },
+  },
+  dashboard: {
+    activateWindowsLicence: {
+      location: PageLocation.tile,
+      buttonType: ButtonType.button,
+      actionType: 'action',
+      actions: ['activate_windows_licence'],
+    },
+    goToVcdPortal: {
+      location: PageLocation.tile,
+      buttonType: ButtonType.externalLink,
+      actionType: 'navigation',
+      actions: ['go-to-access-vcd_portal'],
+    },
+    goToManageBackup: {
+      location: PageLocation.tile,
+      buttonType: ButtonType.externalLink,
+      actionType: 'navigation',
+      actions: ['go-to-see-manage_backup'],
+    },
+  },
+  datacentreListing: {
+    details: {
+      location: PageLocation.datagrid,
+      buttonType: ButtonType.button,
+      actionType: 'navigation',
+      actions: ['details_datacenter'],
+    },
+  },
+  datacentreDashboard: {
+    goToVcdPortal: {
+      location: PageLocation.tile,
+      buttonType: ButtonType.externalLink,
+      actionType: 'navigation',
+      actions: ['go-to-access-vcd_portal'],
+    },
+  },
+  datacentreCompute: {
```

Comment:
compute is a notion dedicated to datacentre so it's enough. Moreover, you can get directly the value with orderType 
` TRACKING[type].`
```suggestion
  compute: {
```